woocommerce/woocommerce-ios
# [Manual Taxes M2.2] Empty State

Closes: #10662 


## Description

With this PR we implement the empty state for the tax rate selector:

### Changes
- Add image for the empty state to project.
- Add empty state with button below.
- Remove empty state text from the bottom notice.
- Refactor to use the same code when button actions are the same.

## Testing instructions


With a store that has no eligible tax rates (no location). The tax based on setting should be customer shipping or billing address to make the Set New Tax Rate row appear.

1. Go to Orders
2. Tap on +
3. Tap on Set New Tax Rate
4. See the empty state. Make sure that the button opens the tax settings in admin.

## Screenshots

<img src="https://github.com/woocommerce/woocommerce-ios/assets/1864060/ea512db3-b166-40a9-aa79-b325a0958c10" width="375">

---
